test/MockBorrower.sol

```diff
@@ -137,8 +143,11 @@ contract MockBorrower {
         return "";
     }
 
-    function flashBorrow(address asset, uint256 amount) public returns (bytes memory) {
-        return lender.flash(address(loanReceiver), asset, amount, "", this.onFlashLoan);
+    function flashBorrow(address asset, uint256 amount) public returns (bytes memory out) {
+        uint256 startingGas = gasleft();
+        out = lender.flash(address(loanReceiver), asset, amount, "", this.onFlashLoan);
+        uint256 endingGas = gasleft();
```

Comment:
I think it'd be better to use something like https://github.com/marktoda/forge-gas-snapshot

Reply:
I can do that, do you want to block this until I get the time? Happy either way.